CRM/Utils/PagerAToZ.php

```diff
@@ -151,7 +151,7 @@ public static function createLinks(&$query, $sortByCharacter, $isDAO) {
       $qfKey = CRM_Utils_Array::value('qfKey', $query->_formValues);
     }
     if (empty($qfKey)) {
-      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', $this, FALSE, NULL, $_REQUEST);
+      $qfKey = CRM_Utils_Request::retrieve('qfKey', 'String', NULL, FALSE, NULL, $_REQUEST);
```

Comment:
@eileenmcnaughton @seamuslee001 this has caused a regression in my local. On visiting 'Manage Event' page it throws:
```
[24-Jul-2017 18:09:38 Asia/Aden] PHP Fatal error:  Cannot pass parameter 3 by reference in /Users/monish/src/civicrm/CRM/Utils/PagerAToZ.php on line 154
```